packages/api-explorer/src/Doc.jsx

```diff
@@ -481,6 +481,7 @@ class Doc extends React.Component {
         onModeChange={this.onModeChange}
         onSubmit={this.onSubmit}
         operation={this.getOperation()}
+        parameters={this.props.user.parameters}
```

Comment:
`parameters` likely won't always be there so this'll come in as `undefined`. Could maybe write this as:

```suggestion
        parameters={this.props.user?.parameters || {}}
```